README.md

```diff
@@ -1,9 +1,67 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) Completed
-2. [Core Java Apps](./core_java) Completed
-3. [Springboot Trading REST API](./springboot) Completed
-4. [Python Data Analytics](./python_data_analytics) Completed
-5. [Hadoop/Hive](./hadoop) In-progress
-6. [Spark/Scala](./spark) In-progress
-7. [Cloud/DevOps](./cloud_devops) In-progress
+# Habib Mohamed . Jarvis Consulting
 
+I am a University of Windsor graduate in Computer Science. I have worked in various positions as a developer working on front-end, data, and website development. I like to figure out solutions to challenging problems, and to come up with unorthodox means to implement stuff.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, HTML/CSS, Docker
+
+**Competent:** Python, C/C++/C#, Spring, JUnit, Flutter, PHP, JavaScript, Maven
+
+**Familiar:** Subversion, Jenkins, Node.js, React.js, Eclipse
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed/tree/master/linux_sql)]: Utilizes Liunx Bash, Postgresql, and Docker to build a monitoring agent that collects data on Linux machine specifications and performance in a cluster.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed/tree/master/core_java)]:
+      
+  - Twitter App: Incorporates Spring Boot, Maven, Docker, and leverages the Twitter API in order to deliver a command line interface to create, read, and delete tweets.
+  - JDBC App: Sets up data manipulation to a retail database using Java, Maven, Docker and Postgresql.
+  - Grep App: Pipelines information on a target directory for files and lines via Java 8 Streams, along with Docker and Maven, in order to find a given regular expression.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed/tree/master/springboot)]: Demonstrates a trading system Proof of Concept for stock traders. Leverages Spring Boot, Maven, Docker, Postgresql, and the IEX API.
```

Comment:
Designed and developed a trading system ...

write more about design, implementation (technologies), testing, deployment (docker stuff)